bin/extract_ietf_modules.py

```diff
@@ -14,7 +13,8 @@
 # either express or implied.
 
 __author__ = 'Benoit Claise, Eric Vyncke'
-__copyright__ = 'Copyright(c) 2015-2019, Cisco Systems, Inc.,  Copyright The IETF Trust 2019, All Rights Reserved'
+__copyright__ = 'Copyright(c) 2015-2018, Cisco Systems, Inc., Copyright The IETF Trust 2019, All Rights Reserved'
```

Comment:
Probably better to keep the year in Cisco's copyright notice as is. It is a condition of the apache v2 license to retain "all copyright, patent, trademark, and attribution notices". It doesn't matter that the years overlap.

Reply:
okay, I reverted it back